client/src/Components/UserSignUp.js

```diff
@@ -112,6 +146,7 @@ function SignUp() {
             variant="contained"
             color="primary"
             className={classes.submit}
+            onClick={e => setValue(e.target.value)}
```

Comment:
I don't think the submit button needs a setVaue, it needs to call handleSubmit to hook up the function you declared up top.